processor/build.gradle

```diff
@@ -8,6 +8,7 @@ buildscript {
 }
 
 apply plugin: "me.champeau.gradle.jmh"
+apply plugin: "java-test-fixtures"
```

Comment:
It creates a source set for test-support code that tests in this project and others can depend on, but that isn't part of the main project. Since there are now tests that use `TestTracingProvider` and `TestTraceHandle` in both `processor` and `testing`.